armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -107,6 +107,7 @@ adduser --system --group --disabled-login --no-create-home              electrs
 adduser --system --group --disabled-login --home /var/run/avahi-daemon  avahi
 adduser --system --group --disabled-login --no-create-home              prometheus
 adduser --system --group --disabled-login --no-create-home              node_exporter
+adduser --disabled-password --gecos "" hdmi
```

Comment:
I wasn't familiar with the `--gecos` flag. I take it this is to avoid needing to specify user details (name, email, "room number") for a non-system user? Speaking of which, is there a reason `hdmi` shouldn't / couldn't be a `--system` user?

Reply:
When using the `--system` flag, a system user is created non-interactively, with no login shell and a UID < 1000. As the user "hdmi" needs to log in to start the x-server, I decided to create a regular user with this `--gecos` flag (which prevents interactive prompts).

Reply:
For my education, could you explain why having a system user with UID < 1000 prevents starting `X`?

Regardless, perhaps we could add a comment explaining why `--gecos ""` is specified, so that future maintainers will have the context for why we set up the `hdmi` user in this way?

Reply:
I don't really care either way, I just thought a user that logs in should not be a system user. One could argue that "hdmi" is a system user, albeit its login capability. 

On second thought I'll switch it to system user to have a more uniform way of creating the users.  Just need to configure the shell using `chsh -s /bin/bash hdmi`.